Dockerfile

```diff
@@ -45,7 +45,6 @@ RUN pip install \
 
 # Drop in the app code
 COPY app/ /app/
-COPY cortex_conf.yml /
```

Comment:
Is the intent not to have don-bot run by itself?
docker build . will not have local.env variables set. So the image won't have any the cortex_conf variables

Reply:
local.env is in test/configuration/local.env. Test will load the file

Reply:
ok. then should add to the readme what env vars to set if user wants to run it as a standalone. and/or point to the local.env file with instructions.

Reply:
i thought the whole point is not to let customer run it standalone, no? that's why i have local.env only for pass docker build.

Reply:
ok i get that but its confusing to have all those instructions in the readme if this isn't meant to be run as a standalone. At the very least, remove those instructions, point to cortex for instructions, and specify which tag customer should fall back to if they want to use it as standalone